src/content/code/plugin-api/layers/findfeaturesbyids-layer.ts

```
@@ -0,0 +1,35 @@
+    }
+  });
+}
+
+const layerId = "01j90ed9m6bxagb6bvfg4sk49q";
+const featureIds = ["6167fcb5-9564-4c8e-a4d3-d0b419f54ec6", "abae3164-f8b3-42bb-b194-0379ecc4c653"];
+
+// Example call to function with multiple layer IDs
+logFeatures(layerId, featureIds);
+
+//2. Define an array of layer IDs to be searched
+const layerId = "01j90ed9m6bxagb6bvfg4sk49q";
```

Biome

[error] Shouldn't redeclare 'layerId'. Consider to delete it or rename it.

'layerId' is defined here:


(lint/suspicious/noRedeclare)